src/main/java/psi/model/DynamicItem.java

```diff
@@ -0,0 +1,20 @@
+package psi.model;
+
+public class DynamicItem implements Item {
+
+	private String value;
+
+	public DynamicItem(String value) {
```

Comment:
There seems to be no functional difference between the DynamicItem and the StaticItem (except the constructor which is missing from the StaticItem, but the public setter & getter do the same job :)  )
Is this the intention? (or maybe I'm missing something)

Reply:
Yes, this was one of the issues I was thinking of. The difference between them is in the way how they will be displayed. I'm still thinking how the classes are impacting the front-end side.